src/main/java/com/dev/theater/model/User.java

```diff
@@ -0,0 +1,59 @@
+package com.dev.theater.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "user")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;
```

Comment:
If it is not unique therefore I can register two similar users, is it ok?

Reply:
the requirements do not contain this condition